sale_stock_ux/models/sale_order_line.py

```diff
@@ -123,3 +132,91 @@ def _onchange_product_uom_qty(self):
             self.product_uom_qty = self._origin.product_uom_qty
             return {'warning': warning_mess}
         return {}
+
+    @api.depends('qty_delivered', 'qty_returned')
+    def _compute_all_qty_delivered(self):
+        for rec in self:
+            rec.all_qty_delivered = rec.qty_delivered + rec.qty_returned
+
+    @api.multi
+    def _get_delivered_qty(self):
```

Comment:
this method do two things: get delivery qty but also update the qty_returned, so will be great to reflect this info in the method name or docstring